app/scripts/services/ErrorMessageResolver.svc.js

```diff
@@ -40,7 +45,7 @@ angular.module('yourStlCourts').factory('errorMessageResolver', function ($q) {
 
           var validationParam = el.attr('ng-' + errorType);
           if (validationParam === undefined) {
-            validationParam = el.attr('data-ng-' + errorType) || el.attr(errorType);
+            validationParam = el.attr('data-ng-' + errorType) || el.attr(errorType) || el.attr(dasherizeFromCamel(errorType));
```

Comment:
What's this used for? The normal angular pattern is to use dashed properties. And it appears your usage was already dashed so was this just leftover from some previous attempt?

Reply:
Angular converts the dashed version to camel case.  So the "errorType" comes through as camel case.  But when I need to get the el.attr()  I need to convert it back to the dashed version so el.attr() works.  This was a headache to figure out.  Without this, I couldn't pass in the number of municipalities and have the error logic handle it automatically.